docs/canvas/canvas-workpad.asciidoc

```diff
@@ -1,49 +1,68 @@
 [role="xpack"]
 [[canvas-workpad]]
-=== Using the workpad
+=== Create a workpad
 
-Now that you have a workpad with sample data that you can mess with, let’s mess with it. 
-We’ll start out by making a few stylistic changes. 
+When you create a workpad, you'll start with a blank page, or you can choose a workpad that is already started for you. 
 
-. Click the gauge chart in the top left of the workpad (fun fact, these are actually pie charts). 
-Try clicking on the small red slice. If you click the number, you'll notice that
-you’ve actually selected a number element, not the gauge itself. 
-+ 
-This element is now selected. Off to the right, you'll see that the side bar has changed. 
-This is where you can make changes to the element.
-+
-[role="screenshot"]
-image::images/canvas_workpad_edit_style.png[]
+. Open *Canvas*.
 
-. Try changing the color palette. Easy right? Click the Back button in your 
-browser to undo your change.
+. Choose one of the following options:
 
-. Try dragging the chart around the page. Grab the resize handles to make 
-the chart bigger and smaller.
+* To start with a single, blank workpad, click *Create workpad*.
 
-. Now click somewhere off of the element, but not on another element, 
-to deselect it. 
-+
-You will notice the sidebar changing again. You now have 
-access to the page and workpad-level settings. Feel free to change the page 
-background color or rename your workpad.
-
-Speaking of pages, Canvas workpads can have multiple pages.
-
-. Click the arrow to the right of the *Page 1 of 2* text in the toolbar at the bottom
-of the page. You'll see a whole new view of the the eCommerce data.
-. Click the *Page 2 of 2* text to see a live preview of all of the pages in your 
-workpad.
-. Click the (+) button to create a new, blank page.
-. Click the *Add element* button in the top right hand corner of the page.
-. Add an *Area Chart.*
+* To create a workpad from a template, select *Templates*, then select a preconfigured template.
+
+* To import an existing workpad, click and drag a workpad JSON file to the *Import workpad JSON file* field.
+
+For advanced workpad examples, add a {kibana-ref}/add-sample-data.html[sample Kibana data set], then select *Canvas* from the *View Data* drop-down list.
+
+For more workpad inspiration, go to the link:elastic.co[Elastic Blog] where we share Canvas updates and customer stories. 
+
+[float]
+==== Personalize your workpad
+
+To personalize your workpad, use the editor on the right.
+
+. Add a *Name* to your workpad, for example, `My Canvas workpad`.
+
+. In the *Width* and *Height* fields, specify the size of your workpad. 
+
+. Select the layout for your workpad. 
 +
-[role="screenshot"]
-image::images/canvas_workpad_3_page.png[]
+For example, click *720p* for a traditional presentation layout.
+
+. Click the *Background color* picker, then select the background color for your workpad.
```

Comment:
This is just setting the background color for the current page, not for every page in the workpad. If you add another page, it'll have the default white background.

Reply:
Since they have not yet added another page, I don't want to confuse them by telling them to do something before they actually do it. I will definitely add this to the subsequent elements piece.